js/draw.js

```diff
@@ -1,78 +0,0 @@
-function draw() {
-  let darkGreen = color("rgb(17, 40, 21)");
-  background(darkGreen);
-
-  if (toggle == 1) {
-    scrollingText();
-  }
-
-  explosions = explosions.filter((explosion) => {
-    drawExplosion(explosion);
-    return explosion.ttl > 0;
-  });
-
-  player = player.filter((player) => {
-    drawPlayer(player);
-    movement(player);
-    return true;
-  });
-
-  playerShells = playerShells.filter((shell) => {
-    drawPlayerShell(shell);
-    for (let enemy of enemies) {
-      const collision = collisionBetweenTwoRectangles(enemy, shell);
-
-      if (collision) {
-        newExplosion(enemy.x, enemy.y, 65, 65);
-        splatSound.play();
-        splatSound.volume(0.3);
```

Comment:
maybe splatSound.play() and splatSound.volume(0.3) could be put into newExplosion? 

newExplosion also looks a little like new Explosion - this should show you how close we actually are to classes. In general, this might be worth refactoring, to see how both would look. For that I would highly recommned using a separate branch though!